src/cryptonote_core/service_node_rules.cpp

```diff
@@ -158,6 +169,11 @@ uint64_t get_min_node_contribution_in_portions(uint8_t version, uint64_t staking
   return result;
 }
 
+uint64_t get_min_node_operator_contribution(uint64_t staking_requirement)
+{
+  return staking_requirement / MAX_NUMBER_OF_CONTRIBUTORS_V1;
```

Comment:
This calculation feel wrong to me, though the actual *value* it calculates isn't.  It just feels wrong to be depending on a `_V1` constant when we aren't at V1 anymore.  I think instead we should have a new constant for this, something like
`MIN_OPERATOR_SHARE` that is set to 4.